docker/mysql_tests/scripts/base_tests/full_mysqldump_partitial_test.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+set -e -x
+
+. /usr/local/export_common.sh
+
+export WALE_S3_PREFIX=s3://mysqlfullmysqldumpbucket
```

Comment:
use a separate bucket for this test, you can add it [here](https://github.com/wal-g/wal-g/blob/490b27137551f1dbccac4a61ffe27f728e976ad4/docker-compose.yml)